service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java

```diff
@@ -290,6 +290,17 @@ void getAllLocationTest() {
         verify(regionRepository).findAll();
     }
 
+    @Test
+    void getActiveLocationsTest() {
+        List<Region> regionList = ModelUtils.getAllRegion();
+
+        when(regionRepository.findAll()).thenReturn(regionList);
+
+        superAdminService.getActiveLocations();
```

Comment:
please check response too

Reply:
please add a test if `regionRepository.findAll()` returns inactive locations too to check this line of code https://github.com/ita-social-projects/GreenCityUBS/pull/922/files#diff-749434d8995cf471e2c87f8ee3afdf70c2cc5301040766ab9c3b60d2703ae0bfR248